### ISSUE TITLE
Version select element: missing child elements

### ISSUE THREAD

**Issue by MelSumner** [emberjs/guides#2185]
**Context:**
Certain ARIA roles are required to have specific child elements, or be contained in specific parent elements. 
If the role of "combobox" is present, then other child elements must be present (see screenshot). 

**Reference:** 
https://www.w3.org/TR/2017/NOTE-wai-aria-practices-1.1-20171214/examples/combobox/aria1.1pattern/listbox-combo.html 

**Screenshot:** 
![image](https://user-images.githubusercontent.com/4587451/35195356-e5d7b73e-fe87-11e7-9eee-d8504883692d.png)
